data/card_definitions.json

```diff
@@ -22915,6 +22915,363 @@
+					"condition": "not_stunned",
+					"effect_type": "force_for_effect",
+					"per_force_effect": null,
+					"overall_effect": {
+						"effect_type": "sustain_this",
+						"card_name": "Splendid Claw"
+					},
+					"force_max": 1
+				}
+			]
+		}
+	},
+	{
+		"id": "vega_scarletterror",
+		"type": "special",
+		"display_name": "Scarlet Terror",
+		"force_cost": 0,
+		"gauge_cost": 0,
+		"range_min": 1,
+		"range_max": 2,
+		"speed": 6,
+		"power": 3,
+		"armor": 0,
+		"guard": 0,
+		"effects": [
+			{
+				"timing": "during_strike",
+				"condition": "is_critical",
+				"effect_type": "speedup",
+				"amount": 2
+			},
+			{
+				"timing": "before",
+				"effect_type": "close",
+				"amount": 2
+			}
+		],
+		"boost": {
+			"boost_type": "immediate",
+			"force_cost": 2,
+			"cancel_cost": -1,
+			"display_name": "Repel Leap",
+			"effects": [
+				{
+					"timing": "immediate",
+					"condition": "at_edge_of_arena",
+					"effect_type": "vega_ua_guarantee",
+					"and": {
+						"effect_type": "move_to_any_space",
+						"and": {
+							"effect_type": "strike"
+						}
+					}
+				}
+			]
+		}
+	},
+	{
+		"id": "vega_flyingbarcelonaattack",
+		"type": "special",
+		"display_name": "Flying Barcelona Attack",
+		"force_cost": 0,
+		"gauge_cost": 0,
+		"range_min": 1,
+		"range_max": 2,
+		"speed": 3,
+		"power": 5,
+		"armor": 0,
+		"guard": 0,
+		"effects": [
+			{
+				"timing": "during_strike",
+				"condition": "is_critical",
+				"effect_type": "powerup",
+				"amount": 2
+			},
+			{
+				"timing": "before",
+				"effect_type": "advance",
+				"amount": 9,
+				"and": {
+					"effect_type": "close",
+					"amount": 1
+				}
+			}
+		],
+		"boost": {
+			"boost_type": "immediate",
+			"force_cost": 0,
+			"cancel_cost": -1,
+			"display_name": "Wall Leap",
+			"effects": [
+				{
+					"timing": "immediate",
+					"effect_type": "advance",
+					"amount": 9
+				}
+			]
+		}
+	},
+	{
+		"id": "vega_skyhighclaw",
+		"type": "special",
+		"display_name": "Sky High Claw",
+		"force_cost": 0,
+		"gauge_cost": 0,
+		"range_min": 1,
+		"range_max": 1,
+		"speed": 3,
+		"power": 4,
+		"armor": 0,
+		"guard": 0,
+		"effects": [
+			{
+				"timing": "during_strike",
+				"condition": "is_critical",
+				"effect_type": "powerup",
+				"amount": 2
+			},
+			{
+				"timing": "before",
+				"effect_type": "close",
+				"amount": 3
+			}
+		],
+		"boost": {
+			"boost_type": "immediate",
+			"force_cost": 1,
+			"cancel_cost": -1,
+			"display_name": "Retrieve",
+			"effects": [
+				{
+					"timing": "immediate",
+					"effect_type": "choice",
+					"choice": [
+						{ "effect_type": "advance", "amount": 3, "bonus_effect": {
+							"condition": "advanced_through",
+							"effect_type": "choose_discard",
+							"limitation": "continuous",
+							"destination": "play_boost",
+							"amount": 1,
+							"amount_min": 1,
+							"discard_effect": {
+								"effect_type": "boost_then_sustain_topdiscard",
+								"amount": "DISCARDED_COUNT",
```

Comment:
I think we do already have a play_boost effect or something similar (one of propeller knight's boosts plays another boost from discard), in case that makes this any easier to implement

Reply:
ah, good point, i'll switch to that i guess.